tests/acceptance/subscribe/SubscribeCheckoutBlockOnOrderProcessingEventCest.php

```diff
@@ -373,7 +361,7 @@ public function testOptInWhenCheckedWithFormAndSimpleProductWithProductForm(Acce
 				'check_opt_in'              => true,
 				'plugin_form_tag_sequence'  => 'form:' . $_ENV['CONVERTKIT_API_FORM_ID'],
 				'subscription_event'        => 'processing',
-				'product_form_tag_sequence' => 'form:' . $_ENV['CONVERTKIT_API_LEGACY_FORM_ID'],
+				'product_form_tag_sequence' => 'form:' . $_ENV['CONVERTKIT_API_SECOND_FORM_ID'],
```

Comment:
Subscribing to a legacy form ID doesn't currently work with the v4 API.  I'd like to revert this test change if the API can support this in the future.  Question [here](https://n7studios-workspace.slack.com/archives/C02KFR6N1GF/p1717765281687949).